helm/portieris/gencerts

```diff
@@ -2,8 +2,11 @@
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 SERVICE_NAME=$(yq r $SCRIPT_DIR/Chart.yaml name)
+NAMESPACE=${1:-ibm-system}    
```

Comment:
In my opinion, the namespace should be taken from the chart's `values.yaml` file to avoid typos, it could be done like this:

```bash
NAMESPACE=$(yq r $SCRIPT_DIR/values.yaml namespace)
```